cmd/podman/machine/create.go

```diff
@@ -0,0 +1,132 @@
+package machine
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/containers/common/pkg/completion"
+	"github.com/containers/podman/v3/cmd/podman/registry"
+	"github.com/containers/podman/v3/pkg/domain/entities"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	createCmd = &cobra.Command{
+		Use:               "create [options] NAME",
+		Short:             "Create a vm",
+		Long:              "Create a virtual machine for podman to run on. Virtual machines are used to run podman on Macs. ",
```

Comment:
Podman except in commands, sorry for not picking up on this earlier.
```suggestion
		Long:              "Create a virtual machine for Podman to run on. Virtual machines are used to run Podman on Macs. ",
```